core/src/main/java/org/apache/iceberg/deletes/PositionDeleteWriter.java

```diff
@@ -56,7 +55,6 @@ public PositionDeleteWriter(
     this.spec = spec;
     this.partition = partition;
     this.keyMetadata = keyMetadata != null ? keyMetadata.buffer() : null;
-    this.delete = PositionDelete.create();
```

Comment:
Public api?

Reply:
This hasn't been used anymore after the deprecated methods have been removed in the previous PR